src/pages/patientView/timeline/timeline_helpers.tsx

```diff
@@ -125,8 +125,6 @@ export function configureTimelineToxicityColors(baseConfig: ITimelineConfig) {
             const path = e.containingTrack.uid.split('.');
             if (path[0] === 'TREATMENT' && path.length > 2) {
                 return getColor(path[2]);
-            } else {
-                return POINT_COLOR;
```

Comment:
Ideally, we should explicitly do `return undefined` here, so that it's clear that for some cases we don't return a valid value. Right now, I think the function now implicitly returns undefined for this case.